Makefile

```diff
@@ -263,6 +263,8 @@ ifneq ($(findstring centos,$(OSNICK)),)
 VECSIM_MARCH ?= skylake-avx512
 else ifneq ($(findstring xenial,$(OSNICK)),)
 VECSIM_MARCH ?= skylake-avx512
+else ifneq ($(findstring macos,$(OS)),)
+VECSIM_MARCH ?= skylake-avx512
```

Comment:
Curious - what is this

Reply:
It's for building with the best instruction set available (vecsim can support multiple and select the best one at runtime, but gcc does not always have the one you really want on a given platform - at least until we start building our own gcc).